requests_cache/session.py

```diff
@@ -58,7 +59,7 @@ def __init__(
         stale_if_error: Union[bool, int] = False,
         **kwargs,
     ):
-        self.cache = init_backend(cache_name, backend, serializer=serializer, **kwargs)
+        self.cache = init_backend(str(cache_name), backend, serializer=serializer, **kwargs)
```

Comment:
This part isn't necessary; this will be converted to a `Path` ([here](https://github.com/requests-cache/requests-cache/blob/main/requests_cache/backends/sqlite.py#L476)) for SQLite and Filesystem backends.

Instead, could you update the signature for `init_backend()` to include `Path` objects? Thanks!

Reply:
I'll try to get to this in the next few days, thanks!